docs/make.jl

```diff
@@ -27,5 +27,5 @@ makedocs(sitename = "JumpProcesses.jl",
                                   mathengine),
          pages = pages)
 
-deploydocs(repo = "github.com/SciML/JumpProcesses.jl.git";
```

Comment:
Why did you change this?

Reply:
Now we can't preview the build on the PR.

Reply:
I shouldn't have committed this. I just commented out while developing it on my machine.